ksql-api/src/test/java/io/confluent/ksql/api/ApiTest.java

```diff
@@ -108,61 +110,66 @@ public void tearDown() {
-    assertNotNull(queryId);
-    assertFalse(server.getQueryIDs().contains(new PushQueryId(queryId)));
+    assertThat(queryId, is(notNullValue()));
+    assertThat(server.getQueryIDs().contains(new PushQueryId(queryId)), is(false));
     Integer rowCount = queryResponse.responseObject.getInteger("rowCount");
-    assertNotNull(rowCount);
-    assertEquals(DEFAULT_ROWS.size(), rowCount.intValue());
+    assertThat(rowCount, is(notNullValue()));
+    assertThat(rowCount, is(DEFAULT_ROWS.size()));
   }
 
   @Test
   public void shouldExecutePushQuery() throws Exception {
 
+    // When
     QueryResponse queryResponse = executePushQueryAndWaitForRows(DEFAULT_PUSH_QUERY_REQUEST_BODY);
 
-    assertEquals("select * from foo", testEndpoints.getLastSql());
-    assertTrue(testEndpoints.getLastPush());
-    assertEquals(DEFAULT_PUSH_QUERY_REQUEST_PROPERTIES, testEndpoints.getLastProperties());
-
-    assertEquals(DEFAULT_COLUMN_NAMES, queryResponse.responseObject.getJsonArray("columnNames"));
-    assertEquals(DEFAULT_COLUMN_TYPES, queryResponse.responseObject.getJsonArray("columnTypes"));
-    assertEquals(DEFAULT_ROWS, queryResponse.rows);
-    assertEquals(1, server.getQueryIDs().size());
-
+    // Then
+    assertThat(testEndpoints.getLastSql(), is("select * from foo"));
+    assertThat(testEndpoints.getLastPush(), is(true));
+    assertThat(testEndpoints.getLastProperties(), is(DEFAULT_PUSH_QUERY_REQUEST_PROPERTIES));
+    assertThat(queryResponse.responseObject.getJsonArray("columnNames"), is(DEFAULT_COLUMN_NAMES));
+    assertThat(queryResponse.responseObject.getJsonArray("columnTypes"), is(DEFAULT_COLUMN_TYPES));
+    assertThat(queryResponse.rows, is(DEFAULT_ROWS));
+    assertThat(server.getQueryIDs(), hasSize(1));
     String queryId = queryResponse.responseObject.getString("queryId");
-    assertNotNull(queryId);
-    assertTrue(server.getQueryIDs().contains(new PushQueryId(queryId)));
-    assertNull(queryResponse.responseObject.getInteger("rowCount"));
+    assertThat(queryId, is(notNullValue()));
+    assertThat(server.getQueryIDs().contains(new PushQueryId(queryId)), is(true));
+    assertThat(queryResponse.responseObject.getInteger("rowCount"), is(nullValue()));
   }
 
   @Test
   public void shouldExecuteMultiplePushQueries() throws Exception {
 
     int numQueries = 10;
     for (int i = 0; i < numQueries; i++) {
+      // When
       QueryResponse queryResponse = executePushQueryAndWaitForRows(DEFAULT_PUSH_QUERY_REQUEST_BODY);
-      assertEquals(i + 1, server.getQueryIDs().size());
+
+      // Then
+      assertThat(server.getQueryIDs(), hasSize(i + 1));
       String queryId = queryResponse.responseObject.getString("queryId");
-      assertNotNull(queryId);
-      assertTrue(server.getQueryIDs().contains(new PushQueryId(queryId)));
+      assertThat(queryId, is(notNullValue()));
+      assertThat(server.getQueryIDs().contains(new PushQueryId(queryId)), is(true));
```

Comment:
FYI, can be:

```suggestion
      assertThat(server.getQueryIDs(), hasItem(new PushQueryId(queryId)));
```

`hasItem()` asserts that a collection has at least one item that matches that supplied.

There is also `hasItems()` which will look for multiple.

Then there is `contains` and `containsInAnyOrder`: both of these look to assert the exact content of the collection, i.e. all entries in the collection must be matched.